Dockerfile

```diff
@@ -82,8 +82,8 @@ RUN mkdir -p ${APP_ROOT} && \
     curl -L https://github.com/ManageIQ/manageiq-ui-self_service/tarball/${REF} | tar vxz -C ${SSUI_ROOT} --strip 1
 
 ## Add ManageIQ source from local directory (dockerfile development) or from Github (official build)
-# ADD . ${APP_ROOT}
-RUN curl -L https://github.com/ManageIQ/manageiq/tarball/${REF} | tar vxz -C ${APP_ROOT} --strip 1
+ADD . ${APP_ROOT}
```

Comment:
@bazulay thank you for sending that link.  What I'm wondering is that COPY / ADD are similar with one difference is that it can grab URL's and will uncompress it.  As we aren't really doing that, I'm trying to understand why we are using ADD as the recommended usage is COPY.